src/push/index.ts

```diff
@@ -49,24 +49,42 @@ import type { PushOptions, ProjectSettings } from '../common_types';
 import { findSyntheticsConfig, readConfig } from '../config';
 
 export async function push(monitors: Monitor[], options: PushOptions) {
-  if (monitors.length === 0) {
-    throw 'No Monitors found';
-  }
-  const duplicates = trackDuplicates(monitors);
-  if (duplicates.size > 0) {
-    throw error(formatDuplicateError(duplicates));
-  }
+  let schemas: MonitorSchema[] = [];
+  if (monitors.length > 0) {
+    const duplicates = trackDuplicates(monitors);
+    if (duplicates.size > 0) {
+      throw error(formatDuplicateError(duplicates));
+    }
 
-  progress(`preparing all monitors`);
-  const schemas = await buildMonitorSchema(monitors);
+    progress(`preparing all monitors`);
+    schemas = await buildMonitorSchema(monitors);
 
-  progress(`creating all monitors`);
-  for (const schema of schemas) {
-    await pushMonitors({ schemas: [schema], keepStale: true, options });
-  }
+    progress(`creating all monitors`);
+    for (const schema of schemas) {
+      await pushMonitors({ schemas: [schema], keepStale: true, options });
+    }
+  } else {
+    write('');
+    // Makes testing easier with overrides
+    let deleteAll = false;
+    if (process.env.TEST_OVERRIDE != null) {
```

Comment:
I think we should add a `--yes` flag to skip this in case someone wants to automate it for any reasons.

I did try running with `yes | npx @elastic/synthetics push` but our command really dislikes that and tries to process all the input thus stalling.

Reply:
I will address this separately and open a PR.